netci.groovy

```diff
@@ -1007,6 +1007,61 @@ def static getR2RDisplayName(def scenario) {
     return displayStr
 }
 
+def static getScenarioDisplayString(def scenario) {
+    switch (scenario) {
+        case 'innerloop':
+            return "Innerloop Build and Test"
+
+        case 'no_tiered_compilation_innerloop':
+            def displayStr = getStressModeDisplayName(scenario)
+            return "Innerloop Build and Test (Jit - ${displayStr})"
+
+        case 'corefx_innerloop':
+            return "CoreFX Tests"
+
+        case 'normal':
+            return "Build and Test"
+
+        case 'jitdiff':
+            return "Jit Diff Build and Test"
+
+        case 'ilrt':
+            return "IL RoundTrip Build and Test"
+
+        case 'longgc':
+            return "Long-Running GC Build & Test"
+
+        case 'gcsimulator':
+            return "GC Simulator"
+
+        case 'standalone_gc':
+            return "Standalone GC"
+
+        case 'gc_reliability_framework':
+            return "GC Reliability Framework"
+
+        case 'illink':
+            return "via ILLink"
+
+        default:
+            if (isJitStressScenario(scenario)) {
+                def displayStr = getStressModeDisplayName(scenario)
+                return "Build and Test (Jit - ${displayStr})"
+            }
+            else if (isR2RScenario(scenario)) {
+                def displayStr = getR2RDisplayName(scenario)
+                return "${displayStr} Build and Test"
+            }
+            else {
+                return "${scenario}"
+            }
+            break
+    }
+
+    println("Unknown scenario: ${scenario}");
```

Comment:
Shouldn't this be under `default:else`?

Reply:
Well, it's actually just intended to ensure nobody "fell through" instead of returned. Don't really need the println at all, but it might be useful for debugging a processing error. The default else just uses the scenario name, which is what I want.